grr/server/grr_response_server/gui/ui/components/client_details/client_diff.ts

```diff
@@ -202,3 +202,59 @@ export function getClientVersions(clientSnapshots: Client[]):
 
   return clientChanges;
 }
+
+function getFirstStringsJoinedPath(path: any[]): string {
+  let tokens: string[] = [];
+  path.forEach(val => {
+    if (typeof val !== 'string') return;
+    tokens.push(val);
+  });
+
+  return tokens.join('.');
+}
+
+function getPathsOfChangedEntries(differences: Diff<Client, Client>[]):
+    string[] {
+  const pathsSetOfChangedEntries: Set<string> = new Set();
+
+  differences.forEach(diffItem => {
+    if (diffItem.path === undefined) return;
+    const joinedPath = getFirstStringsJoinedPath(diffItem.path);
+    if (!RELEVANT_ENTRIES_LABEL_MAP.has(joinedPath)) return;
+
+    pathsSetOfChangedEntries.add(joinedPath);
+  });
+
+  return [...pathsSetOfChangedEntries.values()];
+}
+
+/**
+ * Returns relevant entry paths mapped to the client snapshots in which the
+ * changes were introduced, reverse chronologically ordered. A property path may
+ * not be inside the map if no changes were applied to it since it's creation
+ * @param clientSnapshots an array of chronologically reverse ordered client
+ *     snapshots
```

Comment:
Provide a one-liner description of what the function does in the first line of JSDoc. Then after a blank line possible further explanations.